src/components/layout/org-admin-dashboard-link.tsx

```diff
@@ -80,6 +75,7 @@ export const OrgAdminDashboardLink: FC<OrgAdminDashboardLinkProps> = ({ isVisibl
                     component={Link}
                     href={`${orgAdminBaseUrl}/settings`}
                     active={pathname === `${orgAdminBaseUrl}/settings`}
+                    aria-current={pathname === `${orgAdminBaseUrl}/settings` ? 'page' : undefined}
```

Comment:
https://mantine.dev/core/nav-link/#active says that the pathname should be taken into account and active be set automatically, is that not occurring?